source/Ship.cpp

```diff
@@ -399,7 +399,6 @@ void Ship::FinishLoading(bool isNewInstance)
 		}
 	}
 	// Recalculate the "isDisabled" flag based on this ship's hull and crew.
-	isDisabled = true;
```

Comment:
`isDisabled` is required to be set `true` before the evaluation of `IsDisabled()` since a full check for the disabled status is desired.